static/js/factories/posts.js

```diff
@@ -14,6 +14,8 @@ export const makePost = (isURLPost: boolean = false): Post => ({
   author: `justareddituser${String(Math.random())}`,
   text: isURLPost ? null : casual.text,
   url: isURLPost ? casual.url : null,
+  created: casual.moment,
```

Comment:
will `moment` objects be ok in the redux store? sometimes things with functions defined on them can get a little wacky (I had trouble in the past with Immutable objects, for instance).